pkg/action/actions.go

```diff
@@ -10,4 +10,14 @@ const (
 	// The ActionRequest.Payload for this action contains a single string entry `namespace` with a value
 	// of the new current namespace.
 	RequestSetNamespace = "action.octant.dev/setNamespace"
+
+	// RequestSteFilter is the action for when the filters are updated in Octant.
+	// The ActionRequest.Payload for this action contains an array of key/values representing
+	// the filters.
+	RequestSetFilter = "action.octant.dev/setFilter"
```

Comment:
This can be consolidated with `RequestAddFilter`:

https://github.com/vmware-tanzu/octant/blob/9b658f64342cb6cb8388f689dc61a39cb79de4e5/internal/api/filter_manager.go#L22

Reply:
I had considered using `setFilter` but it seems like it could be used for any of the filter actions. Setting it for one will not read right. The only caveat if we go that route would be that the front end will be responsible for sending the appropriate aggregate filters list as a payload. Having a specific action name based on what it does reads a bit better. What are your thoughts about this?